components/LocalePicker.php

```diff
@@ -72,6 +77,9 @@ public function onSwitchLocale()
             return;
         }
 
+        // Remember the current locale before switching to the requested one
+        $this->oldLocale = $this->translator->getLocale();
```

Comment:
Is there a reason to store this in an instance variable? Wouldn't a locale variable be enough?

Reply:
@see https://github.com/datune/translate-plugin/blob/4d6d3695016d2a61dda1c777fd7db7d939c21ab6/components/LocalePicker.php#L156

Reply:
yes, I know the original PR but I think a local variable would do. Or give me an example why a public instance variable makes more sense?